src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -0,0 +1,416 @@
+<?php
```

Comment:
@adamzimmermann Did you get a chance to review this file? I just wanted to flag it because GitHub keeps collapsing it by default.